docs/axes/labelling.md

```diff
@@ -9,6 +9,7 @@ The scale label configuration is nested under the scale configuration in the `sc
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | `display` | `boolean` | `false` | If true, display the axis title.
+| `labelAlign` | `string` | `'center'` | Alignment of the axis title. Possible options are `'start'`, `'center'` and `'end'`
```

Comment:
It looks like for legend that we have `options.legend.align`. Maybe this should also be just `align` for consistency? Otherwise we have `scaleLabel.labelAlign` and I'm not sure we need "label" to appear twice in a row. What do you think @kurkle ?

Reply:
I can update it accordingly. @kurkle ?